src/app/modules/ix-forms/services/form-error-handler.service.ts

```diff
@@ -4,13 +4,15 @@ import { ResponseErrorType } from 'app/enums/response-error-type.enum';
 import { Job } from 'app/interfaces/job.interface';
 import { WebSocketError } from 'app/interfaces/websocket-error.interface';
 import { DialogService } from 'app/modules/dialog/dialog.service';
+import { IxFormService } from 'app/modules/ix-forms/services/ix-form.service';
 import { ErrorHandlerService } from 'app/services/error-handler.service';
 
 @Injectable({ providedIn: 'root' })
 export class FormErrorHandlerService {
   constructor(
     private dialog: DialogService,
     private errorHandler: ErrorHandlerService,
+    private ixFormService: IxFormService,
```

Comment:
Usually, we don't use the `ix` prefix inside components if such name is not taken